### ISSUE TITLE
Clue Scrolls: Clue icons on npcs etc should show the appropriate difficulty of clue

### ISSUE THREAD

**Issue by ShaunDreclin** [runelite/runelite#8532]
Since the different difficulties have unique icons now, it would be good to show the proper one

![screenshot](https://i.imgur.com/VaP0mjv.png)

**Comment by deathbeam** [comment]
I dont think this matters at all. E.g what abut dig clues? We show just spade there. These "clue" icons are supposed to indicate interaction, and not clue tier.

**Comment by ShaunDreclin** [comment]
Well the spade ones are fine as is, it just seems a little odd to be holding a beginner clue and seeing the master clue icon haha

More of a flavour/polish thing than a necessity

**Comment by deathbeam** [comment]
It should be then maybe changed to actual interaction icon, e.g some talk symbol, some search symbol on crates etc, that would be better.

**Comment by ShaunDreclin** [comment]
Yeah that'd work too
